test-tap/runner.js

```diff
@@ -330,6 +332,110 @@ test('only test', t => {
 	});
 });
 
+test('snapshots not updated when only test', async t => {
```

Comment:
We should try and implement this as a self-hosted integration test: https://github.com/avajs/ava/tree/master/test

Let me know if you want me to give this a go first.

Reply:
I implemented this here, because it was more simple than creating a snapshot, change the test file and try to update the snapshot again. Here i just supply a different runner-chain.

Reply:
Sure, I'm just hoping we can stop writing TAP tests. Will commit an alternative implementation.